Elements/src/Geometry/Line.cs

```diff
@@ -1045,6 +1045,21 @@ public bool TryGetOverlap(Line line, out Line overlap)
             return true;
         }
 
+        /// <summary>
+        /// Calculate U parameter for point on line
+        /// </summary>
+        /// <param name="point"></param>
+        /// <returns>Returns U parameter for point on line</returns>
+        public double GetUParameter(Vector3 point)
```

Comment:
Could we call this `GetParameterAt`? We have other methods for getting points on lines that have names like `GetPointAt(...)`.

Reply:
done